### PR TITLE
Call control application sdks

### PR DESCRIPTION
**Pull request by rajjy1**

### CODE REVIEW

File: src/Telnyx.net/Entities/Calls/CallControlApplications/WebhookAPIVersion.cs [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    /// </summary>
+    /// <value>Determines which webhook format will be used, Telnyx API v1 or v2.</value>
+    [JsonConverter(typeof(Newtonsoft.Json.Converters.StringEnumConverter))]
+    public enum WebhookAPIVersion
+    {
+        /// <summary>
+        /// Enum _1Enum for 1
+        /// </summary>
+        [EnumMember(Value = "1")]
+        _1Enum = 0,
```
Comment (thedonmon) [inline src/Telnyx.net/Entities/Calls/CallControlApplications/WebhookAPIVersion.cs:17]: Is it possible to rename these to V1 and V2 respectively? Or Version1 Version2

File: src/Telnyx.net/Services/Calls/CallControlApplications/ListCallControlApplicationOptions.cs [hunk 2]
```diff
@@ -0,0 +1,15 @@
+﻿namespace Telnyx.net.Services.Calls.CallControlApplications
+{
+    public class ListCallControlApplicationOptions : ListOptions
+    {
+        public int? PageNumber { get; set; }
```
Comment (thedonmon) [inline src/Telnyx.net/Services/Calls/CallControlApplications/ListCallControlApplicationOptions.cs:5]: these are now in the base ListOptions Class can be removed (PageNumber and PageSize)

File: src/Telnyx.net/Entities/Calls/CallControlApplications/AnchorsiteOverride.cs [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+namespace Telnyx.net.Entities.Calls.CallControlApplications
+{
+    [JsonConverter(typeof(Newtonsoft.Json.Converters.StringEnumConverter))]
+    public enum AnchorsiteOverride
+    {
+        /// <summary>
+        /// Enum LatencyEnum for Latency
+        /// </summary>
+        [EnumMember(Value = "Latency")]
+        LatencyEnum = 0,
```
Comment (thedonmon) [inline src/Telnyx.net/Entities/Calls/CallControlApplications/AnchorsiteOverride.cs:13]: Enum doesnt have to be in the name.

File: src/Telnyx.net/Entities/Calls/CallControlApplications/CallControlApplicationInbound.cs [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+        /// </summary>
+        /// <value>This option can be enabled to receive calls from: \"Anyone\" (any SIP endpoint in the public Internet) or \"Only my connections\" (any connection assigned to the same Telnyx user).</value>
+        [JsonConverter(typeof(Newtonsoft.Json.Converters.StringEnumConverter))]
+        public enum SipSubdomainReceiveSettingsEnum
+        {
+            /// <summary>
+            /// Enum OnlyMyConnectionsEnum for only_my_connections
+            /// </summary>
+            [EnumMember(Value = "only_my_connections")]
+            OnlyMyConnectionsEnum = 0,
```
Comment (thedonmon) [inline src/Telnyx.net/Entities/Calls/CallControlApplications/CallControlApplicationInbound.cs:33]: Enum doesnt have to be in the name

File: src/Telnyx.net/Entities/Calls/CallControlApplications/DTMFType.cs [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+    /// </summary>
+    /// <value>Sets the type of DTMF digits sent from Telnyx to this Connection. Note that DTMF digits sent to Telnyx will be accepted in all formats.</value>
+    [JsonConverter(typeof(Newtonsoft.Json.Converters.StringEnumConverter))]
+    public enum DTMFType
+    {
+        /// <summary>
+        /// Enum RFC2833Enum for RFC 2833
+        /// </summary>
+        [EnumMember(Value = "RFC 2833")]
+        RFC2833Enum = 0,
```
Comment (thedonmon) [inline src/Telnyx.net/Entities/Calls/CallControlApplications/DTMFType.cs:17]: Enum can be removed from the names

File: src/Telnyx.net/Entities/Calls/CallControlApplications/CallControlApplicationInbound.cs [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+        /// </summary>
+        /// <value>When set, this will limit the total number of inbound calls to phone numbers associated with this connection.</value>
+        [JsonProperty("channel_limit")]
+        public int? ChannelLimit { get; set; }
+
+        /// <summary>
+        /// Specifies a subdomain that can be used to receive Inbound calls to a Connection, in the same way a phone number is used, from a SIP endpoint. Example: the subdomain \&quot;example.sip.telnyx.com\&quot; can be called from any SIP endpoint by using the SIP URI \&quot;sip:@example.sip.telnyx.com\&quot; where the user part can be any alphanumeric value. Please note TLS encrypted calls are not allowed for subdomain calls.
+        /// </summary>
+        /// <value>Specifies a subdomain that can be used to receive Inbound calls to a Connection, in the same way a phone number is used, from a SIP endpoint. Example: the subdomain \&quot;example.sip.telnyx.com\&quot; can be called from any SIP endpoint by using the SIP URI \&quot;sip:@example.sip.telnyx.com\&quot; where the user part can be any alphanumeric value. Please note TLS encrypted calls are not allowed for subdomain calls.</value>
```
Comment (thedonmon) [inline src/Telnyx.net/Entities/Calls/CallControlApplications/CallControlApplicationInbound.cs:18]: Side note: <value> doesnt actually work as a comment. Can move to Summary section

File: src/Telnyx.net/Services/Calls/CallControlApplications/ListCallControlApplicationOptions.cs (hunk 2 above)
Comment (rajjy1) [reply src/Telnyx.net/Services/Calls/CallControlApplications/ListCallControlApplicationOptions.cs:5]: Done.

File: src/Telnyx.net/Entities/Calls/CallControlApplications/AnchorsiteOverride.cs (hunk 3 above)
Comment (rajjy1) [reply src/Telnyx.net/Entities/Calls/CallControlApplications/AnchorsiteOverride.cs:13]: Changed.

File: src/Telnyx.net/Entities/Calls/CallControlApplications/CallControlApplicationInbound.cs (hunk 4 above)
Comment (rajjy1) [reply src/Telnyx.net/Entities/Calls/CallControlApplications/CallControlApplicationInbound.cs:33]: changed.

File: src/Telnyx.net/Entities/Calls/CallControlApplications/DTMFType.cs (hunk 5 above)
Comment (rajjy1) [reply src/Telnyx.net/Entities/Calls/CallControlApplications/DTMFType.cs:17]: changed.

File: src/Telnyx.net/Entities/Calls/CallControlApplications/CallControlApplicationInbound.cs (hunk 6 above)
Comment (rajjy1) [reply src/Telnyx.net/Entities/Calls/CallControlApplications/CallControlApplicationInbound.cs:18]: done.

File: src/Telnyx.net/Entities/Calls/CallControlApplications/AnchorsiteOverride.cs [hunk 7]
```diff
@@ -0,0 +1,69 @@
+﻿using Newtonsoft.Json;
```
Comment (thedonmon) [inline src/Telnyx.net/Entities/Calls/CallControlApplications/AnchorsiteOverride.cs:1]: Can we move this file into a more common folder? Looks like this and WebHookEnum will be used in more than one service (working on connections now). I was going to put it under Entities -> Enum. Be sure to change the namespace

File: src/Telnyx.net/Entities/Calls/CallControlApplications/WebhookAPIVersion.cs [hunk 8]
```diff
@@ -0,0 +1,24 @@
+namespace Telnyx.net.Entities.Calls.CallControlApplications
```
Comment (thedonmon) [inline src/Telnyx.net/Entities/Calls/CallControlApplications/WebhookAPIVersion.cs:1]: Move to the common Enum folder

File: src/Telnyx.net/Services/Calls/CallControlApplications/CallControlApplicationWritable.cs [hunk 9]
```diff
@@ -0,0 +1,87 @@
+﻿namespace Telnyx.net.Services.Calls.CallControlApplications
+{
+    using Newtonsoft.Json;
+    using Telnyx.net.Entities.Calls.CallControlApplications;
+
+    public class CallControlApplicationWritable : BaseOptions
```
Comment (thedonmon) [inline src/Telnyx.net/Services/Calls/CallControlApplications/CallControlApplicationWritable.cs:6]: Rename this to CreateCallControlApplication or CreateCallControlApplicationOptions

File: src/Telnyx.net/Entities/Calls/CallControlApplications/AnchorsiteOverride.cs (hunk 7 above)
Comment (rajjy1) [reply src/Telnyx.net/Entities/Calls/CallControlApplications/AnchorsiteOverride.cs:1]: Ok. moved to entities > Enum

File: src/Telnyx.net/Entities/Calls/CallControlApplications/WebhookAPIVersion.cs (hunk 8 above)
Comment (rajjy1) [reply src/Telnyx.net/Entities/Calls/CallControlApplications/WebhookAPIVersion.cs:1]: Done.

File: src/Telnyx.net/Services/Calls/CallControlApplications/CallControlApplicationWritable.cs (hunk 9 above)
Comment (rajjy1) [reply src/Telnyx.net/Services/Calls/CallControlApplications/CallControlApplicationWritable.cs:6]: Renamed.

**Pull request merged**